src/components/books/book/book.js

```diff
@@ -1,12 +1,21 @@
-/* eslint-disable react/prop-types */
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../../../redux/books/books';
+import { removeBook, API } from '../../../redux/books/books';
 
 const Book = (props) => {
-  const { name, author, id } = props;
+  const { name, id } = { ...props };
+  console.log(name, ' ', id);
```

Comment:
- [ ] Please remove any console.log statements on your app. This is part of the best practices of JS as it keeps the code clean and professional. [You can read more about Best Practices on JS here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

>Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.

Also, hint: you were having an issue with the Types of the Props, the correct way to fix this issue is using the library PropTypes like this:

![image](https://user-images.githubusercontent.com/79114233/145682706-c56bb8ef-ae42-4f42-ba91-1190ee8df82b.png)

This will help you keep the types of the props on check and will help you debug in the future. [You can read more about it here.
](https://reactjs.org/docs/typechecking-with-proptypes.html)